kubernetes/versi-values.yaml

```diff
@@ -0,0 +1,25 @@
+chain:
+  name: versi
+
+serviceAccount:
+  create: true
+  name: versi
+
+server:
+  secret:
+    # todo secret
+    SMF_BACKEND_FAUCET_ACCOUNT_MNEMONIC: from-gitlab-ci-secret-variables
+  config:
+    SMF_BACKEND_RPC_ENDPOINT: "https://http-versi-rpc-node-0.parity-versi.parity.io"
+    SMF_BACKEND_INJECTED_TYPES: '{}'
+    SMF_BACKEND_NETWORK_DECIMALS: 12
+
+bot:
+  secret:
+    SMF_BOT_MATRIX_ACCESS_TOKEN: from-gitlab-ci-secret-variables
+  config:
+    SMF_BOT_MATRIX_SERVER: 'https://matrix.parity.io'
+    SMF_BOT_MATRIX_BOT_USER_ID: '@versi-faucet:matrix.parity.io'
```

Comment:
should be westend bot? or that's stub?
(other values here and rococo-values.yaml - same question)

Reply:
oops, sorry, I though i was viewing [westend-values.yaml](https://github.com/paritytech/substrate-matrix-faucet/pull/162/files#diff-aecdbe5d882a8aca5911d4d4535ff22969d4896febb7a7ba6d807b570c28eb5d) file.
Disregard this please